sos/plugins/networking.py

```diff
@@ -209,6 +234,25 @@ def setup(self):
             "bridge -d vlan show"
         ])
 
+        # check if devlink module loaded and if yes, capture additional
+        # data from devlink command
+        if self.is_module_loaded("devlink"):
+            self.add_cmd_output("devlink dev show")
+            self.add_cmd_output("devlink port show")
+            self.add_cmd_output("devlink sb show")
+            self.add_cmd_output("devlink sb pool show")
+            self.add_cmd_output("devlink sb port pool show")
+            self.add_cmd_output("devlink sb tc bind show")
+            self.add_cmd_output("devlink region show")
+            self.add_cmd_output("devlink dev param show")
+            for dev in self.get_devlink_devs():
```

Comment:
All the `add_*()` collection methods in `sos` accept either a string with a single item to collect, or a list of strings with multiple items. Please change this to make a single call to `add_cmd_output()` with each `devlink` command as an element in the list argument.